### ISSUE TITLE
Conflicting values for resource 'Files/Microsoft.Maui/Platform/Windows/Styles/WindowRootViewStyle.xbf' 

### ISSUE THREAD

**Issue by garcia252** [dotnet/maui#20592]
### Description

We just upgraded Microsoft.AspNetCore.Components.WebView.Maui and Microsoft.Maui.Controls to 8.0.7. When compiling it gives us the error:

```
2024-02-14T15:31:58.2063992Z WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI175: 0x80073b0f - Processing Resources failed with error: Duplicate Entry. [D:\a\1\s\...MyApp.csproj::TargetFramework=net8.0-windows10.0.19041.0]
2024-02-14T15:31:58.2080335Z WINAPPSDKGENERATEPROJECTPRIFILE : error : PRI277: 0x80073b0f - Conflicting values for resource 'Files/Microsoft.Maui/Platform/Windows/Styles/WindowRootViewStyle.xbf' 
```

No other code changes were made whatsoever. Recently we used 8.0.6 and it compiled fine.

### Steps to Reproduce

Upgrade Microsoft.AspNetCore.Components.WebView.Maui and Microsoft.Maui.Controls to 8.0.7

### Version with bug

8.0.7 SR2

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

8.0.6 SR1

### Affected platforms

Windows

**Comment by SonicScholar** [comment]
This is probably a dupe of the bug that was addressed in [.NET 8.0.201](https://github.com/dotnet/core/blob/main/release-notes/8.0/8.0.2/8.0.201.md)

We had this issue on our Azure pipelines, but the issue is resolved after confirming the update to 8.0.201 took place in our UseDotNet pipeline task.

**Comment by PureWeen** [comment]
> This is probably a dupe of the bug that was addressed in [.NET 8.0.201](https://github.com/dotnet/core/blob/main/release-notes/8.0/8.0.2/8.0.201.md)
> 
> We had this issue on our Azure pipelines, but the issue is resolved after confirming the update to 8.0.201 took place in our UseDotNet pipeline task.

@garcia252 can you confirm?
